nntrainer/tensor/memory_pool.cpp

```diff
@@ -250,6 +250,22 @@ size_t MemoryPool::calcMinMemoryRequirement() {
                         return val1.second < val2.second;
                       });
   unsigned int last_interval = max_interval.second;
+  /** as weights stay valid for max duration, ignore this value and get the real
```

Comment:
Looks like this method needs some dedicated tests later if there isn't any?

Reply:
https://github.com/nnstreamer/nntrainer/pull/1479 added them. Let me know if some cases are missed.

Reply:
Checked, thanks :)